### PR TITLE
receive the events from firestore

### PR DESCRIPTION
**Pull request by farhadbahrehmandhenry**
issue #51

### CODE REVIEW

File: app/src/main/java/com/ad430/globaltactics/EventViewModel.java [hunk 1]
```diff
@@ -1,97 +1,47 @@
...
-            dataMAp4.put("url","http://www.gsmathrive.com/northamerica/");
-            list.add(dataMAp4);
-
-            HashMap<String,String> dataMAp5 = new HashMap<>();
-            dataMAp5.put("Title","November 2020");
-            dataMAp5.put("date","Nov. 30");
-            dataMAp5.put("description","Preparing for the next globalisation: International business after covid-19");
-            dataMAp5.put("url","http://www.gsmathrive.com/northamerica/");
-            list.add(dataMAp5);
+    private final String TAG = OurExpertsViewModel.class.getSimpleName();
```
Comment (lbrown51) [inline app/src/main/java/com/ad430/globaltactics/EventViewModel.java:18]: Is this right?

**Review by lbrown51 — COMMENTED** [review]
Look great

File: app/src/main/java/com/ad430/globaltactics/EventViewModel.java (hunk 1 above)
Comment (farhadbahrehmandhenry) [reply app/src/main/java/com/ad430/globaltactics/EventViewModel.java:18]: hahahahha, that's not definitely right.

**Review by lbrown51 — APPROVED** [review]
Adds features as described and passes test locally and on CircleCI. Great work!

**Pull request merged**